cob_canopen_motor/package.xml

```diff
@@ -21,6 +21,6 @@
   <run_depend>cob_utilities</run_depend>
 
   <export>
-    <cpp lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lcob_canopen_motor_harmonica" cflags="-I${prefix}/common/include"/>
+    <cpp cflags="-I${prefix}/common/include" lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lcob_canopen_motor_harmonica"/>
```

Comment:
Do we need those flags at anymore? I guess this is a leftover from `rosbuild`...